pkg/vpa/vpa.go

```diff
@@ -258,3 +259,28 @@ func (r Reconciler) createVPA(namespace string, vpaName string) error {
 	}
 	return nil
 }
+
+func vpaUpdateModeForNamespace(ns *corev1.Namespace) v1beta2.UpdateMode {
+	// See: https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1beta2/types.go#L101
+	updateMode := v1beta2.UpdateModeOff
+	for k, v := range ns.GetLabels() {
+		if strings.ToLower(k) != vpaUpdateModeLabel {
+			continue
+		}
+		switch v {
```

Comment:
Should we just switch on `strings.ToLower(v)` instead of checking for different capitalizations?

Reply:
Oh, that makes way more sense 🤦‍♂I'll update this

- [ ] Use `strings.ToLower(v)`